packages/bentley/grouping-mapping-widget/src/config/index.ts

```diff
@@ -0,0 +1,21 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+import { Config } from "@bentley/bentleyjs-core";
+
+interface WidgetBuddiConfig {
+  region: string;
+}
+
+export interface WidgetConfig {
+  buddi?: WidgetBuddiConfig;
+}
+
+export const getConfig = (): WidgetConfig => {
```

Comment:
is this really necessary? in your reporting client couldnt you just access Config.app directly?

Reply:
seems like more code from demo-portal that doesnt make sense here

Reply:
This is going to be removed when we switch to 3.0.0. Do you happen to know the replacement for Config.App?

Reply:
IMJS_URL_PREFIX

Reply:
Sorry, I meant to ask where this static singleton was moved to since I cannot find it within @itwin/core-bentley.

Reply:
The singleton has been removed altogether in 3.0, is there some specific piece you need to use?

Reply:
Is there another way to access the region from the core client itself?